components/layout_thread/lib.rs

```diff
@@ -515,6 +521,7 @@ impl LayoutThread {
                 error_reporter: self.error_reporter.clone(),
                 local_context_creation_data: Mutex::new(local_style_context_creation_data),
                 timer: self.timer.clone(),
+                quirks_mode: self.quirks_mode.unwrap_or(QuirksMode::NoQuirks),
```

Comment:
Looking only at this file, I wasn't sure `quirks_mode` would be always already populated when eg `repaint` or `tick_animations` or others would be called. I think it's not a big deal to have a default value here, what do you think ?